builder/googlecompute/config.go

```diff
@@ -149,8 +149,8 @@ type Config struct {
 	// family always returns its latest image that is not deprecated. Example:
 	// "debian-8".
 	SourceImageFamily string `mapstructure:"source_image_family" required:"true"`
-	// The project ID of the project containing the source image.
-	SourceImageProjectId string `mapstructure:"source_image_project_id" required:"false"`
+	// The project ID's of the project containing the source image.
```

Comment:
Just modifying this docstring and running `make generate` updates the documentation for us. So, maybe we change the docstring to read 

```
// A list of project IDs to search for the source image. Packer will search the first
// project ID in the list first, and fall back to the next in the list, until it finds the source image. 
```